tests/e2e/operator/operator_test.go

```diff
@@ -120,12 +133,25 @@ var _ = Describe("Operator", Ordered, func() {
 		})
 
 		It("starts successfully", func() {
-			Eventually(kubectl.GetConditions).
-				WithArguments(namespace, "deployment", deploymentName).
-				Should(ContainElement(resourceAvailable), "Operator deployment is not Available; unexpected Condition")
-			Success("Operator deployment is Available")
+			Eventually(func(g Gomega) {
+				// Wait until the operator deployment is available
+				pods, err := k8sclient.AppsV1().Deployments(namespace).Get(context.TODO(), deploymentName, metav1.GetOptions{})
+				g.Expect(err).ToNot(HaveOccurred(), "Error getting deployment")
```

Comment:
With controller-runtime's client, this becomes:
```
deploy := &appsv1.Deployment{}
g.Expect(k8sclient.Get(context.TODO(), key(deploymentName, namespace), deploy)).To(Succeed())
```

Reply:
Yes, but this will only get the pod and check that no error happens, and in this step we need to check the conditions. I'll try to check if is not too complex by using WithTransform